src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/prevalue/blocklist.blockconfiguration.html

```diff
@@ -0,0 +1,30 @@
+<div
+    class="umb-block-list-block-configuration"
+    ng-controller="Umbraco.PropertyEditors.BlockList.BlockConfigurationController as vm"
+    >
+
+    <div
+        class="umb-block-card-grid"
+        ui-sortable="vm.sortableOptions"
+        ng-model="model.value"
+        >
+
+        <umb-block-card
+            block-config-model="block"
+            element-type-model="vm.getElementTypeByKey(block.contentTypeKey)"
+            ng-repeat="block in model.value"
+            ng-class="{'--isOpen':vm.openBlock === block}"
+            ng-click="vm.openBlockOverlay(block)">
+            <div class="__actions">
+                <button type="button" class="btn-reset __action umb-outline" ng-click="vm.requestRemoveBlockByIndex($index); $event.stopPropagation();">
```

Comment:
Well, we can make a method that wraps the request-method together with the $event.stopPropagation();. But I'm not sure I see the need for that. The stopPropagation call is really only necessary because of the specific usage here, cause otherwise, it would trigger the drag n' drop.

The requestRemoveBlockByIndex method is more general and can be used by code, the reason for the name "request" is because the method shows a dialog, that then has to get an approval from the user before the removal is happening.

let me know if you still have a requested based on this info :-)

Reply:
there is also a method without request in front, which does not ask the user but just goes ahead and does it... hope that helps with the understanding

Reply:
OK yes a bit more clearer now I know that the request means **prompt user** as I had assumed the name was todo with network requests instead. Perhaps consider promptRemove.... @nielslyngsoe ?